ompi/mca/btl/usnic/btl_usnic_module.c

```diff
@@ -1205,7 +1269,7 @@ usnic_send(
         /* assign length */
         sseg->ss_len = sizeof(opal_btl_usnic_btl_header_t) + frag->sf_size;
 
-        sseg->ss_channel = USNIC_PRIORITY_CHANNEL;
+        sseg->ss_channel = USNIC_DATA_CHANNEL;
```

Comment:
So before this change, all sends were going over the priority channel (is that for acks?)